rockcraft/parts.py

```diff
@@ -76,6 +76,20 @@ def __init__(
         except craft_parts.PartsError as err:
             raise PartsLifecycleError(str(err)) from err
 
+    def clean(self) -> None:
+        """Remove lifecycle artifacts.
+
+        :param part_names: The names of the parts to clean. If not
```

Comment:
feels a bit awkward to have this `:param` here if the method doesn't take parameters

Reply:
Whoops. Deleted the argument and documentation remained.